packages/shared/src/styles/themes.scss

```diff
@@ -23,6 +23,11 @@
     --text-weight-bold: bold;
     --text-weight-bolder: bolder;
 
+    //Text Align
+    --text-weight-left: start;
+    --text-weight-right: right;
+    --text-weight-center: center;
```

Comment:
Should these be `--text-align-left`, `right` and `center`? Text weight left looks a bit odd to me.

Reply:
oh it's a mistake